pkg/discovery/dtofactory/general_builder.go

```diff
@@ -164,6 +165,7 @@ func (builder generalBuilder) getSoldResourceCommodityWithKey(entityType metrics
 		// This is better then separately posting the capacity into metrics sync
 		// and then reading it here.
 		commSoldBuilder.Capacity(100)
+		commSoldBuilder.UtilizationThresholdPct(vcpuUtilThreshold) // keeping throttling low
```

Comment:
The changes look good. Can you update the corresponding unit test for this additional field?

Reply:
Yes, updated!